### PR TITLE
chore(Sicilia Vola): [IASV-7] Create SiciliaVola voucher generation workunit

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR add the workunit needed for the voucher generation flow.

## List of changes proposed in this pull request
- Added workunit saga
- Added workunit first screen `CheckStatusRouterScreen.tsx`
- Added navigation action

https://user-images.githubusercontent.com/11773070/121690831-9f4ea980-cac6-11eb-8c72-8001ecc7e6ce.mov

## How to test
Activate the feature flag `svEnabled`.
Dispatch the `svGenerateVoucherStart` action to start the workunit.
To agevolate the test of the back and the cancel action there already are two buttons in the `CheckStatusRouterScreen.tsx` screen.

### CODE REVIEW

File: ts/features/bonus/siciliaVola/saga/orchestration/voucherGeneration.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    cancel: svGenerateVoucherCancel,
+    failure: svGenerateVoucherFailure
+  });
+}
+
+/**
+ * This saga handles the SV activation workflow
+ */
+export function* handleSvVoucherGenerationStartActivationSaga(): SagaIterator {
+  yield call(withResetNavigationStack, svVoucherGenerationWorkUnit);
```
Comment (fabriziofff) [inline ts/features/bonus/siciliaVola/saga/orchestration/voucherGeneration.ts:38]:
```suggestion
  const sagaExecution = () =>
    withFailureHandling(() =>
      withResetNavigationStack(svVoucherGenerationWorkUnit)
    );
  yield call(sagaExecution);
```
If you want the automatic handling for failures with the generic screen. If you intend to custom handling the failures in a next implementation, please ignore this suggestion!

**Review by fabriziofff — CHANGES_REQUESTED** [review]

File: ts/features/bonus/siciliaVola/saga/orchestration/voucherGeneration.ts (hunk 1 above)
Comment (debiff) [reply ts/features/bonus/siciliaVola/saga/orchestration/voucherGeneration.ts]: You're right. Updated in [4ddd528](https://github.com/pagopa/io-app/pull/3128/commits/4ddd5283d71ea7f9d8e6603ee4b7feac0cce3981)

File: ts/navigation/WalletNavigator.ts [hunk 2]
```diff
@@ -168,11 +175,20 @@ const cgnConfigMap = cgnEnabled
...
   : {};
 
+const svConfigMap = svEnabled
+  ? {
+      [SV_ROUTES.MAIN]: {
+        screen: SvNavigator
+      }
+    }
+  : {};
+
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/navigation/WalletNavigator.ts]: Thank you, updated in [0723fbe](https://github.com/pagopa/io-app/pull/3128/commits/0723fbe4e28c5fdc33b577b32e1156c44b0d1522)

File: ts/features/bonus/siciliaVola/navigation/routes.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const SV_ROUTES = {
+  MAIN: "SV_ROUTES_MAIN",
+  VOUCHER_GENERATION: {
+    CHECK_STATUS: "CHECK_STATUS"
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/features/bonus/siciliaVola/navigation/routes.ts]: Sure, modified in [3d4f3a0](https://github.com/pagopa/io-app/pull/3128/commits/3d4f3a02bc9c8321ed0a4254edbc85f432e1dac8)

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**